<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KareemYasser" failures="1" tests="6" name="Login" time="334.645" errors="0" timestamp="2022-05-17T12:15:50 EET">
  <testcase classname="baimsLogin.Login" name="ValidGmail" time="51.061"/>
  <testcase classname="baimsLogin.Login" name="InvalidGmail" time="30.573"/>
  <testcase classname="baimsLogin.Login" name="InvalidGPassword" time="40.871"/>
  <testcase classname="baimsLogin.Twitter_Login" name="ValidTwitter" time="55.379">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;https://auth.baims.com/ar&amp;gt; but was:&amp;lt;https://baims.com/&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<https://auth.baims.com/ar> but was:<https://baims.com/>
at baimsLogin.Twitter_Login.ValidTwitter(Twitter_Login.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- ValidTwitter -->
  <testcase classname="baimsLogin.Twitter_Login" name="InvalidTMmail" time="33.84"/>
  <testcase classname="baimsLogin.Twitter_Login" name="InvalidTPassword" time="41.988"/>
</testsuite> <!-- Login -->
